cli/tools/lint.rs

```diff
@@ -228,7 +228,12 @@ pub fn print_rules_list(json: bool, maybe_rules_tags: Option<Vec<String>>) {
     // so use `println!` here instead of `info!`.
     println!("Available rules:");
     for rule in lint_rules.iter() {
-      println!(" - {}", rule.code());
+      print!(" - {}", rule.code());
+      if rule.tags().is_empty() {
+        println!();
+      } else {
+        println!(" [{}]", rule.tags().join(", "))
+      }
```

Comment:
Can you add some flair here? Maybe put the rule name in `colors::cyan` and all the remaining info (tag, help and link) in `colors::gray`?

Reply:
looks like this now
![image](https://github.com/denoland/deno/assets/22427111/795561cd-d612-4b16-97a2-e09169b443ea)